pom.xml

```diff
@@ -91,14 +91,14 @@
     <org-apache-derby-version>10.15.2.0</org-apache-derby-version>
     <osgi-version>6.0.0</osgi-version>
     <qpid-proton-version>0.34.1</qpid-proton-version>
-    <qpid-jms-version>1.9.0</qpid-jms-version>
+    <qpid-jms-version>2.2.0</qpid-jms-version>
```

Comment:
This is effectively applying a downgrade, since 2.3.0 was the effective 'jakarta equivalent' of the pre-existing 1.9.0 dep. The latest release now is 2.4.0.

Reply:
qpid-jms updated to v2.4.0